src/main/proto/netflix/titus/titus_base.proto

```diff
@@ -109,6 +109,23 @@ message RetryPolicy {
     }
 }
 
+/// Token bucket policies
+message TokenBucketPolicy {
+    message FixedIntervalRefillStrategy {
+        uint64 numberOfTokensPerInterval = 3;
```

Comment:
I believe this is normally called the refill rate.

Reply:
We use this name internally, so I wanted to keep it consistent. I am ok with changing it.